### ISSUE TITLE
interface to DDE problems in DifferentialEquations.jl

### ISSUE THREAD

**Issue by dmew** [j-fu/VoronoiFVM.jl#17]
The interface to ODE solvers in DifferentialEquations.jl is fantastic.
What would be really great is to have an interface for DDE problems as well.
Is it feasible or too much to ask for? Thanks!

**Comment by j-fu** [comment]
Yeah that was just a shot into the mist in  the sense - let's try and see if it works, and it seems to do quite well. In particular it allows to get access to state  of  the art  time integrators with minimal implementation cost.

I already put non-autonomous systems (e.g. driven by boundary oscillations) onto my todo list.

DDEs are not on my list so far, but I will think about them.  Do you have a particular problem in mind ?
